charts/ccd-data-store-api/values.yaml

```diff
@@ -60,3 +60,6 @@ java:
     HTTP_CLIENT_SECONDS_IDLE_CONNECTION: '120'
```

Comment:
can we also update Kiran's hardcoded `caseworker-caa` access in here: https://github.com/hmcts/ccd-data-store-api/blob/0c76fbf1e44304b059d46da7e9c669f6047210da/src/main/java/uk/gov/hmcts/ccd/domain/service/common/DefaultEndpointAuthorisationService.java#L14